backend/main/sheets_api/SheetsAPIHandler.java

```diff
@@ -81,15 +90,41 @@ public static Sheets getServiceInstance() {
         return serviceInstance;
     }
 
-    public static void createSheet(String title) throws IOException {
-        Spreadsheet spreadsheet = new Spreadsheet()
-                .setProperties(new SpreadsheetProperties()
-                .setTitle(title));
-            spreadsheet = getServiceInstance().spreadsheets().create(spreadsheet)
-                    .setFields("spreadsheetId")
+    public void createSpreadsheet(String title) {
+        if (spreadsheetId == null) {
+            try {
+                Spreadsheet spreadsheet = new Spreadsheet()
+                        .setProperties(new SpreadsheetProperties()
+                                .setTitle(title));
+                spreadsheet = getServiceInstance().spreadsheets().create(spreadsheet)
+                        .setFields("spreadsheetId")
+                        .execute();
+                spreadsheetId = spreadsheet.getSpreadsheetId();
+            } catch (IOException e) {
+                System.out.println("There was an issue creating the spreadsheet: " + e);
+                System.exit(1);
+            }
+        } else {
+            System.out.println("We already have a spreadsheet we're working with");
+        }
+    }
+
+    // TODO: right now this creates a new sheet and doesn't rename the original sheet created
+    // TODO: We need to ensure that if we're creating two sheets we rename the first one and then create a new one
+    public void createSheet(String name) {
+        try {
+            List<Request> requests = new ArrayList<>();
+            requests.add(new Request()
+                    .setAddSheet(new AddSheetRequest()
+                            .setProperties(new SheetProperties().setTitle(name))));
+            BatchUpdateSpreadsheetRequest requestBody = new BatchUpdateSpreadsheetRequest()
+                    .setRequests(requests);
+            BatchUpdateSpreadsheetResponse response = getServiceInstance().spreadsheets()
+                    .batchUpdate(spreadsheetId, requestBody)
```

Comment:
If we allow the user to edit an existing spreadsheet, we'll need to have a get spreadsheet id method